lib/ansible/plugins/filter/to_nice_json.yml

```diff
@@ -39,6 +39,10 @@ DOCUMENTATION:
       description: If V(True), keys that are not basic Python types will be skipped.
       default: False
       type: bool
+    sort_keys:
```

Comment:
The fact that it works might be unintended per a few lines below:
> 'These parameters to C(json.dumps) will be ignored, they are overridden for internal use: I(cls), I(default), I(indent), I(separators), I(sort_keys).'

Reply:
Since this has been working for a long time so this functionality should be kept.